src/shrutiS/assignment19/ReverseStringArray.java

```diff
@@ -0,0 +1,30 @@
+/*Program : 2 
+reverse given string array.
+input : {"Credits", "Techno","From","Diwali","Happy"}
+output : {"Happy","Diwali","From","Techno","Credits"}*/
+
+package shrutiS.assignment19;
+
+public class ReverseStringArray {
+
+	void reverseStringArray(String[] input) {
+		String[] temp = new String[input.length];
+		int count = 0;
+		System.out.print("\nReverse of given String Array : ");
```

Comment:
as you are not return reverse of array  & you are printing array's word here only, then I think you should not create new array and filling value into that array.